When set `npmClient: "npm"` in lerna.json, throw Unsupported URL Type "network:": network:31879

**What Renovate type are you using?**
Renovate CLI and Renovate GitHub App.

**Describe the bug**

Renovatebot misidentify `lernaClient`.

https://github.com/renovatebot/renovate/blob/f334c568a5039f88f72fb99ee25410a19ffbe501/lib/manager/npm/extract/index.js#L134

When set `"npmClient": "npm"` in `lerna.json`, renovatebot identify `lernaClient` as `yarn`.

As as results, renovatebot invoke npm with unknown options like `--mutex network:31879` and throw foolowing errors.

```

lerna ERR! Error: Command failed: npm install --ignore-scripts --ignore-engines --ignore-platform --mutex network:31879
lerna ERR! npm ERR! code EUNSUPPORTEDPROTOCOL
lerna ERR! npm ERR! Unsupported URL Type "network:": network:31879
```

**Did you see anything helpful in debug logs?**
```
warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
warning jest > jest-cli > jest-environment-jsdom > jsdom > left-pad@1.3.0: use String.prototype.padStart()
lerna info version 2.9.0
lerna info Bootstrapping 3 packages
lerna info lifecycle preinstall
lerna info Installing external dependencies
lerna ERR! execute callback with error
lerna ERR! Error: Command failed: npm install --ignore-scripts --ignore-engines --ignore-platform --mutex network:31879
lerna ERR! npm ERR! code EUNSUPPORTEDPROTOCOL
lerna ERR! npm ERR! Unsupported URL Type "network:": network:31879
lerna ERR! 
lerna ERR! npm ERR! A complete log of this run can be found in:
lerna ERR! npm ERR!     /tmp/renovate-cache/others/npm/_logs/2019-07-05T06_56_27_106Z-debug.log
lerna ERR! 
lerna ERR!     at Promise.all.then.arr (/home/ubuntu/.npm/lerna@2.9.0/lib/node_modules/lerna/node_modules/execa/index.js:236:11)
lerna WARN complete Waiting for 2 child processes to exit. CTRL-C to exit immediately.
{ Error: Command failed: npm install --ignore-scripts --ignore-engines --ignore-platform --mutex network:31879
npm ERR! code EUNSUPPORTEDPROTOCOL
npm ERR! Unsupported URL Type "network:": network:31879
```

This is example Pull Request.

https://github.com/azu/npm-lerna-renovatebot-test_2/pull/7

**To Reproduce**
Steps to reproduce the behavior:

Reproduce repository: https://github.com/azu/npm-lerna-renovatebot-test_2

1.  Set `"npmClient": "npm"` in `lerna.json`
2.  Run renovatebot

**Expected behavior**

Identify `lernaClient` as npm.
Install the dependencies npm + lerna.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Additional context**


Workaround:

- Delete `npmClient` from `lerna.json`

Because, `npmClient` is `npm` by default.

--- a/lib/manager/npm/extract/index.js
+++ b/lib/manager/npm/extract/index.js
@@ -131,7 +131,8 @@ async function extractPackageFile(content, fileName, config) {
   if (lernaJson) {
     lernaDir = path.dirname(fileName);
     lernaPackages = lernaJson.packages;
-    lernaClient = lernaJson.npmClient || lockFiles.yarnLock ? 'yarn' : 'npm';
+    lernaClient =
+      lernaJson.npmClient === 'yarn' || lockFiles.yarnLock ? 'yarn' : 'npm';
   }
 
   const depTypes = {

--- a/test/manager/npm/extract/index.spec.js
+++ b/test/manager/npm/extract/index.spec.js
@@ -137,6 +137,34 @@ describe('manager/npm/extract', () => {
       );
       expect(res).toMatchSnapshot();
     });
+    it('finds "npmClient":"npm" in lerna.json', async () => {
+      platform.getFile = jest.fn(fileName => {
+        if (fileName === 'lerna.json') {
+          return '{ "npmClient": "npm" }';
+        }
+        return null;
+      });
+      const res = await npmExtract.extractPackageFile(
+        input01Content,
+        'package.json',
+        defaultConfig
+      );
+      expect(res).toMatchSnapshot();
+    });
+    it('finds "npmClient":"yarn" in lerna.json', async () => {
+      platform.getFile = jest.fn(fileName => {
+        if (fileName === 'lerna.json') {
+          return '{ "npmClient": "yarn" }';
+        }
+        return null;
+      });
+      const res = await npmExtract.extractPackageFile(
+        input01Content,
+        'package.json',
+        defaultConfig
+      );
+      expect(res).toMatchSnapshot();
+    });
     it('finds complex yarn workspaces', async () => {
       platform.getFile = jest.fn(fileName => {
         if (fileName === 'lerna.json') {

